Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -152,11 +145,51 @@ private static void ShowPlayerResult(Dictionary<string, int> playerResult)
                 WriteColoredMessage($" { playerResults.Key} -> {playerResults.Value}", color);
             }
         }
+        private static void StartGames()
+        {
+            int gameNumber = 1;
+            string repeatGame = "y";
+
+            while(repeatGame == "y")
+            {
+                WriteAndSeperateText($"================================= GAME #{gameNumber} =================================", addNewLineBeforeText:true, addNewLineAfterText:true);
+
+                Dictionary<string, int> playerResult = PlayGame();
+
+                ShowPlayerResult(playerResult);
+
+                repeatGame = AskRepeat();
```

Comment:
`AskRepeat` method name could be changed to `ConfirmGameRepeat`:
- `Confirm` is more more programmatic word to use in code
- It is not entirely clear what to repeat so GameRepeat would add context to it